### ISSUE TITLE
[Feature] Containerisation of the application 

### ISSUE THREAD

**Issue by mahimairaja** [capjamesg/visionscript#5]
Using DevOps tools like Docker to containerize the application, would help the developers to easily access the `visionscript` irrespective of their hardware and software.

**Comment by capjamesg** [comment]
How would this work? We'd have a Docker that runs the Notebook server, or that can be accessed via the CLI to use the language?

**Comment by mahimairaja** [comment]
Yeah James exactly, we could host the Notebook server inside a docker

**Comment by capjamesg** [comment]
Feel free to submit a PR if you are interested. This would be great to have!

**Comment by rexdivakar** [comment]
I would love to work on it, but i dont understand how to app runs as it keeps crashing on my macbook. Kindly advice so i can work on this issue!

**Comment by mahimairaja** [comment]
Hi @rexdivakar,

if you are interested in working on this

I can help you

**Comment by rexdivakar** [comment]
ok cool, can we connect over discord or do you have any other means of chat?

**Comment by mahimairaja** [comment]
Yeah, sure

My discord username - mahimairaja
